### PR TITLE
Test and env updates

### PR DESCRIPTION
**Pull request by ShellyKa13**
**What this PR does / why we need it**:
This change is a first step in trying to make our tests usable to test compatibility with kubevirt for other backup partners.
This change marks tests that apply example manifests with label `PartnerComp`. This allows to run only those tests with gingko filter:
make TEST_ARGS="--test-args=-ginkgo.label-filter=PartnerComp" test-functional
Also those yamls were hardcoded with a storageclass, now changed the framework in order to make it possible to config other SC by updating env variable KVP_STORAGE_CLASS
To apply the yamls manually you'll need to replace the placeholder {{KVP_STORAGE_CLASS}} with your desired storageclass or run it with a replace command such as:
cat tests/manifests/vm_with_pvc.yaml | sed 's/{{KVP_STORAGE_CLASS}}/rook-ceph-block/g' | kubectl create -f -

Updated kubevirtci due to a bug in the last version of a problem running with docker.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Add PartnerComp label
```

### CODE REVIEW

File: velero-v1.12.0-linux-amd64/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (mhenriks) [inline velero-v1.12.0-linux-amd64/LICENSE:1]: Did you mean to add this dir?

**Comment by mhenriks** [comment]
@ShellyKa13 Can you edit the PR description to briefly state reasoning for changes to to tests/manifests?

File: velero-v1.12.0-linux-amd64/LICENSE (hunk 1 above)
Comment (ShellyKa13) [reply velero-v1.12.0-linux-amd64/LICENSE:1]: this was done automatically when running the updated kubevirtci..

Comment (mhenriks) [reply velero-v1.12.0-linux-amd64/LICENSE:1]: Yeah at some point velero is downloaded but I don't think we want to add it to the repo.  Maybe put something in gitignore?

Comment (ShellyKa13) [reply velero-v1.12.0-linux-amd64/LICENSE]: @mhenriks add the gitignore and deleted an existing velero repo we had in here.

**Pull request merged**